requirements.txt

```diff
@@ -10,4 +10,4 @@ python-dotenv==0.14.0
 dateparser~=1.2.0
 pandas~=2.1.4
 numpy>=1.22.2 # not directly required, pinned by Snyk to avoid a vulnerability
-clinical_etl@git+https://github.com/CanDIG/clinical_ETL_code.git@v1.2.1
+clinical_etl@git+https://github.com/CanDIG/clinical_ETL_code.git@develop
```

Comment:
I'd prefer using a version number here: I forgot to make a release for date intervals, but I just did it [here](https://github.com/CanDIG/clinical_ETL_code/releases/tag/v2.0.0) so this should be v2.0.0.